web/src/server/api/routers/generations/db/getAllGenerationsSqlQuery.ts

```diff
@@ -79,7 +81,8 @@ export async function getAllGenerations({
           FROM
             scores
           WHERE
-            project_id = ${input.projectId}
+          project_id = ${input.projectId}
+          AND scores."data_type" IN ('NUMERIC', 'BOOLEAN')
```

Comment:
why exclude categorical here?

Reply:
Used for filter only, categorical scores loaded in separate query